tests/testsuite/alt_registry.rs

```diff
@@ -415,48 +436,43 @@ or use environment variable CARGO_REGISTRIES_ALTERNATIVE_TOKEN",
 
 #[cargo_test]
 fn publish_to_alt_registry() {
-    registry::alt_init();
-    let p = project().file("src/main.rs", "fn main() {}").build();
-
-    // Setup the registry by publishing a package
-    Package::new("bar", "0.0.1").alternative(true).publish();
+    let _reg = RegistryBuilder::new()
+        .http_api()
+        .http_index()
+        .alternative()
+        .build();
 
-    // Login so that we have the token available
-    p.cargo("login --registry alternative TOKEN").run();
+    let p = project().file("src/main.rs", "fn main() {}").build();
 
     // Now perform the actual publish
-    p.cargo("publish --registry alternative").run();
-
-    validate_alt_upload(
```

Comment:
Can you say why this validation was removed? I would not expect it to be removed.

Reply:
*sigh* I was specifically looking for any still removed and missed this.  Thanks for spotting it!

Reply:
@hi-rustin Can you post a PR to return the `publish_to_alt_registry` test so that it validates the upload?